config_mitie.json

```diff
@@ -1,5 +1,5 @@
 {
-  "backend": "mitie",
+  "backend": "mitie_sklearn",
```

Comment:
ah I see. Let's make another `config_mitie_sklearn.json` . If people want to start out by installing a single dependency - that should work. We should probably throw some warnings if training is started with the MITIE backend and > 5 intents and > 50 intent examples.